staff/YaninaGomezCucchiararo/blog/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Healthy BLOG</title>
+    <link rel="icon" href="images/iconhealthy.png" type="image/png">
+</head>
+
+
+<body>
+    <header>
+        <img src="images/iconhealthy.png" alt="icon healthy" title="Healthy blog">
+        <h1>HEALTHY BLOG</h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="">About us</a>
```

Comment:
Why are you using "<a href"">?? i think is not appropriate because it has any meaning, you could add a link or "mailto", that would be much better.